lib/extension/groups.ts

```diff
@@ -167,7 +167,15 @@ export default class Groups extends Extension {
             if (this.state.exists(device)) {
                 const state = this.state.get(device);
                 const endpointNames = device.isDevice() && device.getEndpointNames();
-                const stateKey = endpointNames && endpointNames.length >= member.ID ? `state_${endpointNames[member.ID - 1]}` : 'state';
+                const stateKey =
+                    endpointNames &&
+                    endpointNames.length >= member.ID &&
+                    device.definition &&
+                    device.definition.meta &&
+                    device.definition.meta.multiEndpoint &&
```

Comment:
```suggestion
                    device.definition?.meta?.multiEndpoint &&
```

Should allow you to get rid of:


```
                    device.definition &&
                    device.definition.meta &&
```

Reply:
Change merged.